<launch>

  <node pkg="line_detection" name="backproject_lanedetection" type="backproject.py" output="screen">
    
    <param name="use_compressed_format" type="bool" value="true" />
    <param name="subscriber_image_topic" type="str" value="/camera/image_raw" />
    <param name="publisher_image_topic" type="str" value="/line_image" />
    <param name="training_file_name" type="str" value="training_for_backprojection_1.png" />
    <param name="buffer_size" type="int" value="52428800" />
    <param name="histogram_dimension" type="int" value="50" />

  </node>

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />
  
</launch>